flink-table/flink-table-planner/src/test/java/org/apache/flink/table/planner/functions/TimeFunctionsITCase.java

```diff
@@ -432,15 +435,17 @@ private Stream<TestSetSpec> temporalOverlapsTestCases() {
     }
 
     private Stream<TestSetSpec> ceilTestCases() {
+        TimeZone.setDefault(TimeZone.getTimeZone("UTC"));
         return Stream.of(
                 TestSetSpec.forFunction(BuiltInFunctionDefinitions.FLOOR)
                         .onFieldsWithData(
                                 // https://issues.apache.org/jira/browse/FLINK-17224
```

Comment:
@jnh5y Now, we test TIMESTAMP_TLZ, which means when we do the test, the result depend on the the timezone you run the test, so we should fix the timezone, so each people can pass these tests in their timezone. If I don't fix the time zone, these test can pass in my laptop, but cannot pass in flink ci server, because my time zone is different than flink ci server's time zone

Reply:
we can not make any assumption about flink ci timezone.
Nobody guarantees that together with movement to GitHub it stays same

Reply:
that was the thing I initially tried to suggest
however need to be sure that after test stream finishes the timezone should be reset to the initial value